mail-mta/sendmail/sendmail-8.17.1.9.ebuild

```diff
@@ -20,12 +20,13 @@ fi
 
 LICENSE="Sendmail GPL-2" # GPL-2 is here for initscript
 SLOT="0"
-KEYWORDS="~alpha amd64 arm hppa ~ia64 ~mips ppc ppc64 ~riscv ~s390 sparc x86"
```

Comment:
I don't get why you've modified the old version here. Also, I'm pretty sure I merged sendmail without tinycdb?

It shouldn't be in the same commit as the other stuff for sure at least.

Reply:
> Also, I'm pretty sure I merged sendmail without tinycdb?

I was running into build errors with both versions about a missing `<cdb.h>` file when `dev-db/cdb` is installed and noticed it was provided by tinycdb, unless it's possible to disable cdb support. `dev-db/cdb` conflicts with it I believe since it provides a `libcdb.a` and sendmail automatically enables cdb support if it finds a `libcdb.a` or `libcdb.so`, only one of those are provided by tinycdb. Should a patch be made instead to remove the check for `libcdb.a`?

Reply:
I think so, yeah, but do it in just the new version. Don't touch the stable one.